plugins/TimingTriggerCandidateMaker.hpp

```diff
@@ -55,6 +55,17 @@ class TimingTriggerCandidateMaker : public dunedaq::appfwk::DAQModule
 
   std::string m_hsievent_receive_connection;
 
+  // HSI Passthrough changes
+  std::atomic<bool> m_hsi_passthrough;
+  int hsi_pt_before;
+  int hsi_pt_after;
+  std::bitset<16> trigger_bitmask;
+  std::vector< std::bitset<8> > LowHighBits;
+  std::bitset<16> MakeBitmask16(uint16_t signal_map);
```

Comment:
DAQ style is `snake_case` for function names, so `make_bitmask16`, `make_bitmask8`, `split_bits` and `any_bit_set` (but this comment is moot if you use the alternative method for checking high bits that I suggested above)

Reply:
yup.